lua/lazyvim/plugins/extras/util/gh.lua

```diff
@@ -0,0 +1,63 @@
+return {
+
+  -- depends on the git extra for highlighting and auto-completion of github issues/prs
+  { import = "lazyvim.plugins.extras.lang.git" },
+
+  -- gh.nvim
+  {
+    "ldelossa/gh.nvim",
+    dependencies = {
+      {
+        "ldelossa/litee.nvim",
+        config = function()
+          require("litee.lib").setup()
+        end,
+      },
+    },
+    config = function()
+      require("litee.gh").setup()
+    end,
```

Comment:
```suggestion
    config = function(_, opts)
      require("litee.gh").setup(opts)
    end,
```
This way if a user wants to change default settings, he can directly change them in `opts` in his personal configuration without needing to overwrite the `config` function.